app/services/hulu_mapping_service.rb

```diff
@@ -0,0 +1,83 @@
+# Service to sync Hulu API with Kitsu
+class HuluMappingService
+  HULU_URL = 'https://distribution.hulu.com/api/v1/'.freeze
+
+  attr_reader :since
+
+  # Initialize service with optional datetime
+  def initialize(since = nil)
+    @since = since
+  end
+
+  # Grabs all available series from hulu
+  def get_series_for_episodes(limit)
```

Comment:
This name seems like a pretty long name, is there a reason for the `for_episodes` part?